<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_point</class>
 <widget class="QDialog" name="edit_point">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="Point">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>411</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Données du point :</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="nom_point">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>30</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="point_name_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>36</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom :</string>
    </property>
   </widget>
   <widget class="QLabel" name="categorie_point_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>66</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Categorie :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="categorie_point">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>60</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lat_point_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>97</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Latitude :</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="lat_point">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>91</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Lon_point_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>127</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Longitude : </string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="lon_point">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>122</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="decription_point_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>desciption :</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="description_point">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>154</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="categorie_point_but">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter une categorie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>106</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Position actuel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="heures_point_box">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>401</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Horaires du point :</string>
   </property>
   <widget class="QTableView" name="viewheures">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="jour_heur_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Jour :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="jour_heur">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>225</y>
      <width>261</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="plage_heur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Plage horaire  :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jout_heur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter </string>
    </property>
   </widget>
   <widget class="QPushButton" name="supr_heur">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_heure">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>mettre à jour</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="debut_heur">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>257</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fin_heur">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>257</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Points existant :</string>
   </property>
   <widget class="QPushButton" name="supr_point">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="jout_point">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter </string>
    </property>
   </widget>
   <widget class="QTableView" name="table_point">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>641</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_point">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>metre à jour</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
